src/__tests__/MobileMenu.spec.tsx

```diff
@@ -1,7 +1,46 @@
 import {mount, shallow} from 'enzyme';
 import React from 'react';
+import {MenuItem} from '../Header/MainMenu';
 import {MobileMenu} from '../Header/MobileMenu';
 
+const headerLinkOptions = [
```

Comment:
It's very strange to see copies of the options, links and render method. Maybe it make sense to export them from the original file and test originals? I think it would be even better to place them into some additional file, something like we do with constants